### DISCUSSION TITLE
Update values in data loader through slider input in js block in md file possible?

### DISCUSSION THREAD

**Discussion by gganapat** [observablehq/framework#1087] [Q&A] [answered]
As title suggests, is it possible for data loader values to be refreshed with slider input in .md file calling the data loader? That would be super cool to get interactive plots for example.

gani-

**Comment by mbostock** [comment, marked as answer]
No, data loaders are designed to run at build time, so they cannot respond to reactive inputs. As an alternative, you can either have your data loader generate data for all possible input parameters and then filter on the client, or you can switch to using client-side `fetch` so that your data is not precomputed statically.

**Reply by gganapat** [reply]
Ok, thanks.
G-
